src/cgnslib.h

```diff
@@ -806,6 +839,7 @@ CGNSDLL int cg_configure(int what, void *value);
 CGNSDLL int cg_error_handler(void (*)(int, char *));
 CGNSDLL int cg_set_compress(int compress);
 CGNSDLL int cg_get_compress(int *compress);
+CGNSDLL int cg_set_filter(const struct CG_FILTER *filter);
```

Comment:
I should probably introduce a new cg_set_chunk API, cg_configure is used currently for chunking.

Reply:
added